### DISCUSSION TITLE
How to enable GPU during transcription in Python rather than in shell?

### DISCUSSION THREAD

**Discussion by FYNIXqwq** [openai/whisper#1520] [Q&A]
I'm planning to write a program in Python letting me drag the file and starts the transcription immediately by GPU. This is convenient in shell by ```--device cuda```. However, I could not find a detailed guide of utilization in Python.

**Comment by FYNIXqwq** [comment]
I'm trying
```
model = whisper.load_model("medium")
result = model.transcribe("100.mp4", fp16 = False)
print(result["text"])
```
and the program starts to run on CPU, which is slow.

**Comment by phineas-pta** [comment]
`whisper.load_model(..., device="cuda")`
